src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,79 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { Filter } from '../../services/enums';
+import { TodosContext } from '../../TodosContext';
+import { Todo } from '../../services/types';
+
+function countActiveTodos(todos: Todo[]): number {
+  return todos.filter(todo => {
+    return !todo.completed;
+  }).length;
+}
```

Comment:
create constant like 
```
const activeTodos = todos.filter(({ completed }) => completed)
```
and add this to your context provider, and pass to values